### ISSUE TITLE
 Invalid api response    when trying to Image.list()

### ISSUE THREAD

**Issue by Lolshtole** [pjpimentel/dots#1]
```
digitalOcean.Image.list()...
digitalOcean.Image.list('application')...
```
gives me  `Invalid api response
`

**Comment by pjpimentel** [comment]
Hey @Lolshtole thanks for reporting! :)
I've just commited and published a new version  #2, could you try again and tell me if it is ok?

**Comment by Lolshtole** [comment]
it's working now, thanks!
